iavl/internal/changeset.go

```diff
@@ -0,0 +1,11 @@
+package internal
+
+import "fmt"
+
+// NOTE: This is a placeholder implementation. We will add the implementation in a future PR.
```

Comment:
do we have an issue to track this?

Reply:
It's all implemented in the aaronc/iavlx branch, but these PRs are chunked out of that for easy review and a small surface area for adding unit tests. There isn't really an issue to track the chunking, but we can create one if it would be helpful.